### ISSUE TITLE
Fix the NAN seconds ago issue

### ISSUE THREAD

**Issue by kjnanda** [rtCamp/rtMedia#1648]
When the user uploads the media or comments on any uploaded media it should show `a few second ago` immediately after it is posted. But it shows something like ` NAN seconds  ago ` currently. 

Please fix this issue.

![Photo___Krupa_Nanda___Media___Profile___krupa_dev5_rt_gw](https://user-images.githubusercontent.com/15173772/81294348-61de4800-908c-11ea-89c2-b361c954cd9a.png)

This issue is present with latest release - V4.6.3

**Comment by ExoGeek** [comment]
I also have this issue.

Regards

**Comment by pooja-muchandikar** [comment]
@SID177 

Cross-checked this issue with the following test environment and works as expected.

```
Test Environment
rtMedia: v4.6.3 pre-release branch
BuddyPress: v6.0.0 ( checked with Nouveau and Legacy template )
Theme: 2020 and 2019
```

**Comment by pooja-muchandikar** [comment]
Tested with `develop` branch and the fix works as expected.

**Comment by pooja-muchandikar** [comment]
Tested with `master` branch and the fix works as expected.

**Comment by pooja-muchandikar** [comment]
Released with`v4.6.4`
